.clang-format

```diff
@@ -10,6 +10,7 @@ AllowShortLoopsOnASingleLine: false
 AlignEscapedNewlines: Left
 AlignArrayOfStructures: None
 PointerAlignment: Right
+InsertBraces: true
```

Comment:
@skarg is this ok to you?

Reply:
Yes, InsertBraces is okay with me; saves me from running clang-tidy. 👍